vcsutils/webhookparser/utils.go

```diff
@@ -0,0 +1,9 @@
+package webhookparser
+
+// optional converts a nil pointer to a zero value
+func optional[T any](t *T) T {
+	if t == nil {
+		return *new(T)
+	}
+	return *t
```

Comment:
@cyrilc-pro we have a function that does the same thing - check `DefaultIfNotNil`.
```suggestion
```